### ISSUE TITLE
GarageDoor Example ?

### ISSUE THREAD

**Issue by SYNiMAK** [Mixiaoxiao/Arduino-HomeKit-ESP8266#36]
Just wondering if there is any chance you could do a garage door opener example? 

P.S. Loving your work!

**Comment by Mixiaoxiao** [comment]
This link [garage.c](https://github.com/maximkulkin/esp-homekit-demo/blob/master/examples/garage/garage.c) may help, define your accessory like it and control/report your accessory by reading the Example01~04.

**Comment by SYNiMAK** [comment]
Awesome! Thank you. Will give it a go

**Comment by jmbwell** [comment]
Here's how I did mine: https://github.com/jmbwell/GargeDoorOpener

**Comment by askarkurymbayev** [comment]
you can make garage doors with a wicket function with a door_lock icon?

**Comment by askarkurymbayev** [comment]
can you make garage doors with a wicket function with a door_lock icon?

**Comment by PierreBier** [comment]
Hello, this is a magnificent project. Thank you!
My Problem is, that i have a RF remote control with only one button to open. After 2 minutes the garage door closes automatically. I don't have access to the sensors.
Is there a way to change the code to auto-close after a delay of 2 minutes?

**Comment by thncode** [comment]
> Hello, this is a magnificent project. Thank you!
> My Problem is, that i have a RF remote control with only one button to open. After 2 minutes the garage door closes automatically. I don't have access to the sensors.
> Is there a way to change the code to auto-close after a delay of 2 minutes?

Yes you can use a timer, I do the same with a parking lot barrier...

**Comment by PierreBier** [comment]
Thank you for your answer. I guess I have to change the part with the magnetic sensor into a timer. But how do I do that?
Every time I change something with the magnetic sensors the program crashes.
